quantum/debounce.h

```diff
@@ -9,11 +9,9 @@
  *
  * @param raw The current key state
  * @param cooked The debounced key state
- * @param num_rows Number of rows to debounce
  * @param changed True if raw has changed since the last call
  * @return true Cooked has new keychanges after debouncing
  * @return false Cooked is the same as before
  */
-bool debounce(matrix_row_t raw[], matrix_row_t cooked[], uint8_t num_rows, bool changed);
-
-void debounce_init(uint8_t num_rows);
+void debounce_init(void);
+bool debounce(matrix_row_t raw[], matrix_row_t cooked[], bool changed);
```

Comment:
Otherwise doc string doesnt apply to the correct function.
```suggestion
bool debounce(matrix_row_t raw[], matrix_row_t cooked[], bool changed);

void debounce_init(void);
```